libpod/container.go

```diff
@@ -293,6 +293,10 @@ type ContainerConfig struct {
 	// namespace
 	// These are not used unless CreateNetNS is true
 	PortMappings []ocicni.PortMapping `json:"portMappings,omitempty"`
+	// NoCreateResolvConf indicates that resolv.conf should not be
+	// bind-mounted inside the container.
+	// Conflicts with DNSServer, DNSSearch, DNSOption.
+	NoCreateResolvConf bool
```

Comment:
Why create a flag, rather then carry it though in DNSOption?

Reply:
I'd prefer not to have this in DNS options - makes it easier to say that I want something to run/not run when this is set. The negative logic is mostly so the default remains false, to prevent older containers from breaking - missing fields decode to False, so all old containers will stop making resolv.conf

Reply:
How about calling it 
UseImageResolvConf

Reply:
I concur on the avoidance of negative names, especially when a lot of the tests below are `!NoCreate...` below.

Reply:
Yes all the double negatives bothers my brain.